### PR TITLE
[BD-21] Fix missing `module_name` argument in CourseWaffleFlag constructor

### PR DESCRIPTION
**Pull request by regisb**
**TL;DR -** Get rid of a few warnings and become forward-compatible with edx-toggles==2.0.0.

**What changed?**

Add `module_name` argument to CourseWaffleFlag constructor. This argument is optional for now, but will soon become mandatory.

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [ ] ~~Translations up to date~~
- [ ] ~~JS minified, SASS compiled~~
- [ ] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [Bd-21](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation)

FIY: @edx/masters-devs-gta

This is ready for review.

cc @robrap

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @regisb! I've created [BLENDED-684](https://openedx.atlassian.net/browse/BLENDED-684) to keep track of it in Jira.   More details are on the [BD-21](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation) project page.


When this pull request is ready, tag your edX technical lead.

**Review by robrap — APPROVED** [review]

**Comment by robrap** [comment]
Oops. A different 2.13.6 snuck in first. This needs to be 2.13.7.

**Comment by regisb** [comment]
v2.13.7 has already been released by https://github.com/edx/edx-ora2/pull/1542/ -- so I guess all we need to do is tag the release, right?

**Comment by robrap** [comment]
@regisb: That is right.  I will coordinate with @jansenk tomorrow to ensure it gets released.  I'm not sure if the release was intentionally delayed for some reason, or just mistakenly forgotten.

**Comment by robrap** [comment]
@regisb: FYI: https://github.com/edx/edx-ora2/releases/tag/2.13.7
